OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -141,7 +141,7 @@ public void Draw()
 			for (var i = 0; i < renderables.Count; i++)
 				renderables[i].Render(this);
 
-			// added for contrails
+			// Added for contrails and WeatherOverlay
```

Comment:
Please just remove this comment.  The engine code shouldn't be referring to traits defined in the mod code.